src/pages/settings/user/UserDetails.tsx

```diff
@@ -112,10 +119,11 @@ export function UserDetails() {
         onClose={setPasswordConfirmModalOpen}
         onSave={onSave}
       />
+
       <Tabs tabs={tabs} className="mt-6" />
 
       <div className="my-4">
-        <Outlet />
+        <Outlet context={errors} />
```

Comment:
Instead of using TabGroup and passing errors through components I researched a little bit about passing values through Outlet component. So, this is significantly easier way of passing values to child components instead of plain props. But, let me know about your thoughts.

Reply:
Sure, looks good :)